<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ClientUI</class>
 <widget class="QDialog" name="ClientUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>257</width>
    <height>310</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ClientUI</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label1">
       <property name="text">
        <string>Host Addr:</string>
       </property>
       <property name="buddy">
        <cstring>addr</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="addr"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="laber2">
       <property name="text">
        <string>Host Port:</string>
       </property>
       <property name="buddy">
        <cstring>port</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="port"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>昵称:</string>
       </property>
       <property name="buddy">
        <cstring>nickname</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="nickname"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>抢队（不一定成功）</string>
       </property>
       <property name="buddy">
        <cstring>comboBox</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTextBrowser" name="board"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QPushButton" name="connectButton">
       <property name="text">
        <string>Connect</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="startButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Start</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>addr</tabstop>
  <tabstop>port</tabstop>
  <tabstop>connectButton</tabstop>
  <tabstop>startButton</tabstop>
  <tabstop>cancelButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
